CLI/Repl.cpp

```diff
@@ -444,7 +458,32 @@ int main(int argc, char** argv)
         return 0;
     }
 
-    if (argc >= 2 && strcmp(argv[1], "--compile") == 0)
+    if ((argc >= 2 && strcmp(argv[1], "--compile=text") == 0) || strcmp(argv[1], "--compile") == 0)
+    {
```

Comment:
This needs something like  `_setmode(_fileno(stdout), O_BINARY);` when _WIN32 is defined, see 
https://stackoverflow.com/questions/23107609/is-there-way-to-set-stdout-to-binary-mode